src/cmd/servicePush.go

```diff
@@ -161,7 +173,7 @@ func servicePushCmd() *cmdBuilder.Cmd {
 				[]uxHelpers.Process{{
 					F:                   uxHelpers.CheckZeropsProcess(deployProcess.Id, cmdData.RestApiClient),
 					RunningMessage:      i18n.T(i18n.PushRunning),
-					ErrorMessageMessage: i18n.T(i18n.PushRunning),
+					ErrorMessageMessage: "error",
```

Comment:
tady bych dal nějaké todo aby se tam udělal překlad

Reply:
Or you can add the translation right away